packages/core/build/constants.js

```diff
@@ -124,6 +126,11 @@ const MINIMIZERS = !IS_RELEASE
 
 const plugins = ({ base, is_test_env }) => {
     return [
+        new Dotenv({}),
+        new DefinePlugin({
+            'process.env.DATADOG_APPLICATION_ID': JSON.stringify(process.env.DATADOG_APPLICATION_ID),
+            'process.env.DATADOG_CLIENT_TOKEN': JSON.stringify(process.env.DATADOG_CLIENT_TOKEN),
```

Comment:
shouldn't we add `DATADOG_SESSION_SAMPLE_RATE` and `DATADOG_SESSION_REPLAY_SAMPLE_RATE` here?

Reply:
done